source/_includes/puppet-collections/_puppet_collections_intro.md

```diff
@@ -2,6 +2,15 @@
 > -   For Puppet 3.8 open source packages, see its [repository documentation](/puppet/3.8/puppet_repositories.html).
 > -   For Puppet Enterprise installation tarballs, see its [installation documentation](/pe/latest/install_basic.html).
 
-Puppet maintains official package repositories for several operating systems and distributions. To make the repositories more predictable, we version them as "Puppet Collections" --- each collection has all of the software you need to run a functional Puppet deployment, in versions that are known to work well with each other. Each collection is opt-in, and if you use `ensure => latest` to manage your Puppet packages, you’ll get the latest compatible versions in the collection you’re using. Whenever we make significant breaking changes that introduce incompatibilities between versions of our software, we make a new collection so that you can perform safe major upgrades at your convenience.
+Puppet maintains official package repositories for several operating systems and distributions. To make the repositories more predictable, we version them as "Puppet Collections" --- each collection has all of the software you need to run a functional Puppet deployment, in versions that are known to work well with each other.
 
-We maintain Puppet Collection repositories throughout the corresponding operating systems' lifespan, plus one additional month.
\ No newline at end of file
+The collections are organized into two tiers that map to the Puppet Enterprise release trains which are "downstream" from the open-source components in the collection. Numbered collections (like PC1) are long-lived, stable repositories that map to Long Term Support (LTS) Puppet Enterprise releases; the standard support PE releases will track a collection named `latest` ([The Puppet Enterprise Lifecycle](https://puppet.com/misc/puppet-enterprise-lifecycle) page has more detail about the PE release trains). A collection repository will support operating system-specific packages for the OSes available when it's launched, until thirty days past the end of their vendor-determined lifespan.
```

Comment:
Does this imply LTS for puppetserver and puppetdb as well as puppet-agent? Where will releases of those go?